samples/graphene-hello/preload/manifest.json

```diff
@@ -9,6 +9,9 @@
         "hello": {
             "Package": "world",
             "Parameters": {
+                "Env": {
+                    "HELLO_WORLD": "Hello World from Marblerun!"
+                },
                 "Argv": [
                     "foo",
```

Comment:
we should probably remove this

Reply:
Wouldn't it make sense to have env vars in both of the examples, not only just one?

Reply:
Absolutely. I wanted to comment on line 16, but it's hard to see the difference.